src/Components/AllCountries/AllCountries.js

```diff
@@ -27,55 +27,55 @@ const AllCountries = () => {
           continent: country.continents,
         });
       }
-
-      let filteredCountryList = [];
-      switch (selectedContinent) {
-        case "all":
-          setCountryList(countryData);
-          break;
-        case Continents.Asia:
-          filteredCountryList = countryData.filter(
-            (country) => country.continent[0] === Continents.Asia
-          );
-          setCountryList(filteredCountryList);
-          break;
-        case Continents.Africa:
-          filteredCountryList = countryData.filter(
-            (country) => country.continent[0] === Continents.Africa
-          );
-          setCountryList(filteredCountryList);
-          break;
-        case Continents.Europe:
-          filteredCountryList = countryData.filter(
-            (country) => country.continent[0] === Continents.Europe
-          );
-          setCountryList(filteredCountryList);
-          break;
-        case Continents.NorthAmerica:
-          filteredCountryList = countryData.filter(
-            (country) => country.continent[0] === Continents.NorthAmerica
-          );
-          setCountryList(filteredCountryList);
-          break;
-        case Continents.SouthAmerica:
-          filteredCountryList = countryData.filter(
-            (country) => country.continent[0] === Continents.SouthAmerica
-          );
-          setCountryList(filteredCountryList);
-          break;
-        case Continents.Australia:
-          filteredCountryList = countryData.filter(
-            (country) => country.continent[0] === Continents.Australia
-          );
-          setCountryList(filteredCountryList);
-          break;
-        case Continents.Oceania:
-          filteredCountryList = countryData.filter(
-            (country) => country.continent[0] === Continents.Oceania
-          );
-          setCountryList(filteredCountryList);
-          break;
-      }
+      setCountryList(countryData);
+      // let filteredCountryList = [];
```

Comment:
Remove the comments. If this code is required in future, add a comment describing the reason for comment.